src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveStreams.Core.verified.txt

```diff
@@ -3637,7 +3647,7 @@ namespace Akka.Streams.Implementation
     }
     public class static StreamLayout
     {
-        public static readonly bool IsDebug;
+        public const bool IsDebug = false;
```

Comment:
Made this a compile-time constant since it's only used for debugging Akka.Streams during development on the source itself.